### PR TITLE
fix: Retain data for an extra day for `package_analyses` table.

### PR DESCRIPTION
**Pull request by rafiu007**

### CODE REVIEW

File: f8a_report/helpers/report_helper.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def cleanup_db_tables(self):
                                'package_analyses', 'started_at', num_days)
 
         # Number of days to retain the package analyses data
-        num_days = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '30')
+        num_days = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '31')
```
Comment (prashbnair) [inline f8a_report/helpers/report_helper.py:143]: Any reason for this change?

Comment (rafiu007) [reply f8a_report/helpers/report_helper.py:143]: I looked at the logs of the job for the last 4 days and it passed for the last 2 days but failed for the days before that.From this i concluded that the time at which data is getting stored might be the reason.

Comment (prashbnair) [reply f8a_report/helpers/report_helper.py:143]: But how does changing it by one day help?

Comment (rafiu007) [reply f8a_report/helpers/report_helper.py:143]: As the task runs daily the maximum difference can be that one day.

**Review by prashbnair — APPROVED** [review]

**Pull request merged**